controllers/topics-controller.js

```diff
@@ -0,0 +1,12 @@
+// const db = require('./db/connection')
+const { getAllTopics } = require('../models/topics')
+
+exports.getTopics = (req,res) => {
+    return getAllTopics().then(data => {
+        console.log(data.rows)
+        res.send(data.rows)
```

Comment:
The status will default to 200 so no explicit need to use `res.status(200)`
I would however like to suggest that instead of responding with an array, respond with an object, with a key of topics in this case - on that key is the array